framework/db/mysql/QueryBuilder.php

```diff
@@ -81,6 +81,25 @@ public function renameColumn($table, $oldName, $newName)
     }
 
     /**
+     * Builds a SQL statement for creating a new index.
+     * @param string $name the name of the index. The name will be properly quoted by the method.
+     * @param string $table the table that the new index will be created for. The table name will be properly quoted by the method.
+     * @param string|array $columns the column(s) that should be included in the index. If there are multiple columns,
+     * separate them with commas or use an array to represent them. Each column name will be properly quoted
```

Comment:
`@inheritdoc` could be used instead of copying the whole method doc. Also it would be cool to have a link to MySQL bug here.